projects/shepherd/tsconfig.lib.json

```diff
@@ -19,7 +19,7 @@
   },
   "angularCompilerOptions": {
     "annotateForClosureCompiler": true,
-    "enableIvy": false,
+    "compilationMode": "partial",
```

Comment:
Can you please explain this change? Sorry I do not know a lot about angular.

Reply:
https://blog.angular.io/upcoming-improvements-to-angular-library-distribution-76c02f782aa4